src/journey-maps/angular/components/level-switch/level-switch-horizontal.ts

```diff
@@ -73,15 +73,24 @@ export class SbbLevelSwitchHorizontal implements OnInit, OnDestroy {
   toggleSideButtons(): void {
     this.showSideButtons = !this.showSideButtons;
```

Comment:
Das hier wäre dann auch obsolet, wenn es in den beiden Unter-Methoden gesetzt wird.